tests/pragmas/thintprocessing.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+  matrix: "--hint:processing"
+  nimout: '''
+..
```

Comment:
```suggestion
compile start
..
```

Reply:
shoot, this fails on windows:
```
....compile start
```

(known bug IIRC)

given that we already have a similar test in trunner.nim, maybe add `disabled: windows`?

refs:
```
  block: # `hintProcessing` dots should not interfere with `static: echo` + friends
    let cmd = fmt"""{nim} r {defaultHintsOff} --hint:processing -f --eval:"static: echo 1+1""""
    let (outp, exitCode) = execCmdEx(cmd, options = {poStdErrToStdOut})
    template check3(cond) = doAssert cond, $(outp,)
    doAssert exitCode == 0
    let lines = outp.splitLines
    check3 lines.len == 3
    when not defined(windows): # xxx: on windows, dots not properly handled, gives: `....2\n\n`
      check3 lines[0].isDots
      check3 lines[1] == "2"
      check3 lines[2] == ""
    else:
      check3 "2" in outp
```